rocksdb_replicator/replicated_db.cpp

```diff
@@ -63,10 +63,20 @@ DEFINE_int32(replicator_replication_mode, 0,
              "stack for the connection to one of the Slave; "
              "2: ack client once committed to Master and one of the Slaves.");
 
-DEFINE_uint64(replicator_timeout_ms, 5 * 1000,
-              "How long to wait for Slave before timeout a client write, 0 means"
+DEFINE_uint64(replicator_timeout_ms, 2 * 1000,
+              "How long to wait for Follower ACK before timeout a client write, 0 means"
               " waiting forever");
 
+DEFINE_uint64(replicator_timeout_degraded_ms, 10,
+              "In Degradation mode, i.e. when the writes keep timing out waiting for follower ACK, "
+              "we would use this timeout to fail fast. "
+              "Note that for this config to be valid and used, it needs to be "
+              "smaller than replicator_timeout_ms, but not smaller than 1 (set by kMinReplTimeoutMs)");
```

Comment:
I'm still a little confused.
If value < 0 then this feature is disabled.
if 0 < value < kMinReplTimeoutMs, then also this feature is disabled, right?
Can we make it simpler by saying that this value should be greater than kMinReplTimeoutMs but lesser than replicator_timeout_ms?

Reply:
it's an unsigned integer. reworded a bit, hopefully it's clearer now